metadata-service/restli-client/src/main/java/com/linkedin/entity/client/EntityClient.java

```diff
@@ -274,6 +274,12 @@ public <T extends RecordTemplate> Optional<T> getVersionedAspect(@Nonnull String
       @Nonnull Long version, @Nonnull Class<T> aspectClass, @Nonnull Authentication authentication)
       throws RemoteInvocationException;
 
+  EntityResponse getVersionedEntity(@Nonnull final String entityName,
```

Comment:
semantic versioning, not aspect versioning... we need to be careful to delineate here